models/statistic.go

```diff
@@ -62,5 +63,7 @@ func GetStatistic() (stats Statistic) {
 	stats.Counter.HookTask, _ = db.GetEngine(db.DefaultContext).Count(new(HookTask))
 	stats.Counter.Team, _ = db.GetEngine(db.DefaultContext).Count(new(Team))
 	stats.Counter.Attachment, _ = db.GetEngine(db.DefaultContext).Count(new(Attachment))
+	stats.Counter.Project, _ = db.GetEngine(db.DefaultContext).Count(new(Project))
+	stats.Counter.ProjectBoard, _ = db.GetEngine(db.DefaultContext).Count(new(ProjectBoard))
```

Comment:
While working on it, could you refactor this to
```go
e := db.GetEngine(db.DefaultContext)
stats.Counter.Comment, _ = e.Count(new(Comment))
...
stats.Counter.Project, _ = e.Count(new(Project))
stats.Counter.ProjectBoard, _ = e.Count(new(ProjectBoard))
```

Reply:
Done